SUPPORTED_SCENARIOS.md

```diff
@@ -58,3 +58,7 @@ The goal of this document is to have a single source of information  which scena
 * /boot
 * extended partition
   * /boot has to be on a logical partition
+  
+# Required software
+
+* During installation, yast-bootloader requires installing the packages kexec_tools and grub2. If one decides for not installing grub in MBR <b>and</b> "/" then the system can be installed without these packages.
```

Comment:
scenarios in fact depends on configuration, so it is more like "shim if secure boot is used" but it is only for grub2efi and only on x86_64...so in general I suggest to split it to those three and then check code in which case what is installed. Logic is quite straighforward and can be found in `def packages` methods.